scripts/infra/setup-aws.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env -S bash -e
+# SPDX-License-Identifier: Apache-2.0
+
+ensure_system_pkgs() {
+    # shellcheck source=/dev/null
+    source /etc/os-release
+    if [[ "$ID" == "fedora" ]] || [[ "$ID_LIKE" =~ "fedora" ]]; then
+        PKG_LIST=(git gcc python-devel python-virtualenv krb5-devel openldap-devel aws)
+        sudo dnf install "${PKG_LIST[@]}" -y
+    else
+        echo "neither system 'ID' nor 'ID_LIKE' is 'fedora' - did you mean to specify a 'rh' system?"
+    fi
+}
+
+rh() {
+    local cmdname=$1; shift
+    if [ "$(type -t "rh__$cmdname")" = "function" ] >/dev/null 2>&1; then
```

Comment:
Doesn't seem to be used.  Although perhaps it should be since this method is rh specific.

Reply:
I figured this might be good to have already here for future additions